CVE-2022-25237/cvex.yml

```diff
@@ -0,0 +1,7 @@
+blueprint: ubuntu2204
```

Comment:
Use the ubuntu2204-ubuntu2204 blueprint. Install Bonita on ubuntu1, copy the exploit to ubuntu2, run the exploit on ubuntu2 that would attack ubuntu1. Make sure that router_raw.pcap captures the network traffic. The goal is to get the traffic!